drainer/config.go

```diff
@@ -67,9 +67,10 @@ type SyncerConfig struct {
 	IgnoreTables      []filter.TableName `toml:"ignore-table" json:"ignore-table"`
 	TxnBatch          int                `toml:"txn-batch" json:"txn-batch"`
 	LoopbackControl   bool               `toml:"loopback-control" json:"loopback-control"`
-	SyncDDL           bool               `toml:"sync-ddl" json:"sync-ddl"`
+	SyncDDL           string             `toml:"sync-ddl" json:"sync-ddl"`
```

Comment:
Why use `string`?

Reply:
it extends logic of sync ddl, to support truncate-only scenario.

Reply:
It will cause compatibility issues.